MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,541 @@
+---
+layout: post
+title: Toolbar Customization in .NET MAUI Image Editor Control | Syncfusion
+description: Learn here all about toolbar customization support in the Syncfusion .NET MAUI ImageEditor (SfImageEditor) control.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Toolbar Customization in .NET MAUI Image Editor (SfImageEditor)
+
```

Comment:
You can customize the existing toolbar appearance, visibility, position, and orientation of each toolbar and can include new toolbars